### PR TITLE
BAH-2652 | Added changeset to map lab and radiology orders with test …

### PR DESCRIPTION
**Pull request by deepthi-mantena**
…and radiologyimaging procedure classes

### CODE REVIEW

**Review by gsluthra — APPROVED** [review]

File: bahmnicore-omod/src/main/resources/liquibase.xml [hunk 1]
```diff
@@ -2680,6 +2680,24 @@
             insert into order_type_class_map(order_type_id, concept_class_id) values(@order_type_id,@lab_test);
         </sql>
     </changeSet>
+    <changeSet id="bahmni-core-20221206644-bah-2652" author="clinic-config">
```
Comment (angshu) [inline bahmnicore-omod/src/main/resources/liquibase.xml:2683]: Prefer splitting them in separate changeset. And please use preconditions. we shouldn't assume that the required entities exist or any mapping already does not exist.

File: bahmnicore-omod/src/main/resources/liquibase.xml [hunk 2]
```diff
@@ -2680,6 +2680,24 @@
...
+            set @order_type_id = '';
+            set @radiology_order_type_id = '';
+            set @test_class = '';
+            set @radiology_imaging = '';
+
+
+            select order_type_id into @order_type_id from order_type where name = 'Lab Order';
+            select order_type_id into @radiology_order_type_id from order_type where name = 'Radiology Order';
+            select concept_class_id into @test_class from concept_class where name = 'Test';
+            select concept_class_id into @radiology_imaging from concept_class where name = 'Radiology/Imaging Procedure';
```
Comment (angshu) [inline bahmnicore-omod/src/main/resources/liquibase.xml:2695]: Is "Radiology/Imaging Procedure" class already present in OpenMRS? again, split this onto different changeset and guard with precondition.

File: bahmnicore-omod/src/main/resources/liquibase.xml (hunk 1 above)
Comment (deepthi-mantena) [reply bahmnicore-omod/src/main/resources/liquibase.xml]: Updated the change set Angshu please review.

**Review by angshu — APPROVED** [review]
